src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,27 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger LOGGER = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        LOGGER.info("completeOrder method was called. With params userId: {}", userId);
```

Comment:
Let's make this logger message with level `debug`. Info about called methods usually needed only for debug purposes. If in real application we put info about all methods called to the info level it will increase size of the log files dramatically.